get_ioc_usage.py

```diff
@@ -52,12 +63,22 @@ def print_instruments_with_ioc(instrument_configs, ioc_name):
     :param instrument_configs: instrument ioc config dictionary
     :param ioc_name: name of the ioc
     """
-    print(f"Instruments containing IOCs starting with {ioc_name}")
+    print("Instruments containing IOCs starting with {}".format(ioc_name))
```

Comment:
I think we should be able to use f-strings now as we're only using python 3 - it doesn't really matter but if you think it's more readable with or without them I think it's OK to use either.